portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,70 +27,107 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
+ 
 /** Servlet that returns a random quote. */
 @WebServlet("/data")
 public final class DataServlet extends HttpServlet {
+ 
+    /** Declared as class variable so that initialization happens once in init()
+        rather than everytime in doGET() */
```

Comment:
I can see why you'd explain this, given that in a previous review, I'd asked why this was a member on the class.

Even so, most of the time you don't need to add comments explaining why a member is associated with the class, rather than as a local variable.  A better comment here might be to explain what this list will contain (though the variable name is actually pretty good at explaining that, so it's possible you don't actually need a comment here!).